app/views/agent/partials/vatReturnsPartial.scala.html

```diff
@@ -18,10 +18,10 @@
 @import common.MandationStatus
 @import java.time.LocalDate
 
-@(mandationStatus: String)(implicit messages: Messages,appConfig: AppConfig){
-<div class="card">
-    <div class="card-body">
-        <h2 class="heading-medium">@messages("vatReturnsPartial.heading")</h2>
+@(mandationStatus: String)(implicit messages: Messages,appConfig: AppConfig)
+<div class="tile tile-no-border">
+    <div id="vat-returns" class="tile-body">
+        <h2 class="heading-medium push--top">@messages("vatReturnsPartial.heading")</h2>
         @if(mandationStatus==MandationStatus.nonMTDfB){
         <p>@messages("vatReturnsPartial.paragraphNonMandated")</p>
         <ul>
```

Comment:
Could you add a tiny bit of custom CSS just to add some vertical spacing between the list items? In the prototype it has `margin-bottom: 5px` and makes the links stand out a bit better when there is both `Submit VAT Return` and `View submitted returns` links displayed